todo_app/data/trello_items.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import os
+
+from todo_app.item import Item
+
+auth = {
+      'key': os.getenv('TRELLO_KEY'), 
+      'token': os.getenv('TRELLO_TOKEN'),
+    }
+
+def get_items():
+    response = requests.get('https://trello.com/1/boards/{board}/cards'.format(board=os.getenv('TRELLO_BOARD')), params=auth)
+    cards = response.json()
+    items = []
+    for card in cards:
+        item = Item.from_trello_card(card, map_list_id_to_name(card['idList']))
+        items.append(item)
+    return items
+
+def map_list_id_to_name(listID):
+    lists = get_lists()
+    for list in lists:
+        if listID == list['id']:
+            return list['name']
+    return ''

```

Comment:
Just on your comment re can we make things a bit quicker by reducing the looping - the easiest way I've seen is using the `lists` endpoint mentioned in the exercise with the `cards=open` parameter - that way we get the lists & cards together in one request. Here it might look something like:
```suggestion
def get_items():
    response = requests.get('https://trello.com/1/boards/{board}/lists?cards=open'.format(board=os.getenv('TRELLO_BOARD')), params=auth)
    lists = response.json()
    items = []
    for a_list in lists:
        for card in a_list['cards']:
            item = Item.from_trello_card(card, a_list['name'])
            items.append(item)
    return items
```